### PR TITLE
Change for the final color scheme

### PR DESCRIPTION
**Pull request by GresosJ**

### CODE REVIEW

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]
Update README aswell

**Review by MarioRodrigues10 — COMMENTED** [review]
This looks terrible :-1:

Try to find a way to match colors in a better way

**Comment by GresosJ** [comment]
> 

I like it  a lot

**Comment by MarioRodrigues10** [comment]
I have some concerns about the design and do not feel comfortable approving it at this time. However, I understand that design preferences can vary, so I will defer to the opinions of the other reviewers to make a final decision.

**Review by pedrofp4444 — COMMENTED** [review]
I kind of agree with @MarioRodrigues10... I don't feel like this could be an attractive landing :sweat_smile:. But it's just my opinion, probably I would leave it too to other reviewers :+1:

**Comment by EnzoVieira** [comment]
> I have some concerns about the design and do not feel comfortable approving it at this time. However, I understand that design preferences can vary, so I will defer to the opinions of the other reviewers to make a final decision.

I don't think the design looks terrible. I thinks the design looks very confortable, despite it looks a little bit amateur. Perhaps, we can keep improving design with small changes I would suggest, such as:

Try to do not use the same color on a Title and Subtitle, use a neutral color for the sub to increase the contrast.
Captura de Tela 2023-03-21 às 01 02 30

The same for the dates.
Captura de Tela 2023-03-21 às 01 04 14

Also, the background color of these cards are not a good choice to contrast them, you can simply use the default background color and add a soft shadow box to the card, this way, you keep a simple and modern layout. In this same section, you can change the color of the event hour to a neutral color, the same rule as the title and subtitle.
Captura de Tela 2023-03-21 às 01 04 39

What do you think? @MarioRodrigues10 @GresosJ @pedrofp4444

**Comment by MarioRodrigues10** [comment]
> > I have some concerns about the design and do not feel comfortable approving it at this time. However, I understand that design preferences can vary, so I will defer to the opinions of the other reviewers to make a final decision.
> 
> I don't think the design looks terrible. I thinks the design looks very confortable, despite it looks a little bit amateur. Perhaps, we can keep improving design with small changes I would suggest, such as:
> 
> Try to do not use the same color on a Title and Subtitle, use a neutral color for the sub to increase the contrast. Captura de Tela 2023-03-21 às 01 02 30
> 
> The same for the dates. Captura de Tela 2023-03-21 às 01 04 14
> 
> Also, the background color of these cards are not a good choice to contrast them, you can simply use the default background color and add a soft shadow box to the card, this way, you keep a simple and modern layout. In this same section, you can change the color of the event hour to a neutral color, the same rule as the title and subtitle. Captura de Tela 2023-03-21 às 01 04 39
> 
> What do you think? @MarioRodrigues10 @GresosJ @pedrofp4444

I suggest you to compare with the design that we have now and express your opinion about which one you prefere.

**Comment by EnzoVieira** [comment]
> I suggest you to compare with the design that we have now and express you opinion about which one you prefere.

I think the colors we are using now are not in line with the theme of the event. The purple color doesn't convey any sense. Also, a dark theme is a personal choice, but for a law theme, I think it doesn't make sense. But again, a dark theme is just a personal choice.

**Comment by MarioRodrigues10** [comment]
> > I suggest you to compare with the design that we have now and express you opinion about which one you prefere.
> 
> I think the colors we are using now are not in line with the theme of the event. The purple color doesn't convey any sense. Also, a dark theme is a personal choice, but for a law theme, I think it doesn't make sense. But again, a dark theme is just a personal choice.

How about we keep the design as it is for the light mode, but add a button for people to toggle to a cool purple-themed dark mode? That way, everyone gets to choose what they like.

**Comment by EnzoVieira** [comment]
> > > I suggest you to compare with the design that we have now and express you opinion about which one you prefere.
> > 
> > 
> > I think the colors we are using now are not in line with the theme of the event. The purple color doesn't convey any sense. Also, a dark theme is a personal choice, but for a law theme, I think it doesn't make sense. But again, a dark theme is just a personal choice.
> 
> How about we keep the design as it is for the light mode, but add a button for people to toggle to a cool purple-themed dark mode? That way, everyone gets to choose what they like.

Totally agree with you, but I still don't understand why purple 😅, it's just my opinion, so let's see what the others think!

File: public/images/organization/CeSIUM-icon.svg
Comment (pedrofp4444) [inline public/images/organization/CeSIUM-icon.svg:1]: I think you should use the white lettering. I don't think that this one is readable :sweat_smile: 
gerson

**Review by pedrofp4444 — CHANGES_REQUESTED** [review]

**Comment by JulioJPinto** [comment]
> > > I suggest you to compare with the design that we have now and express you opinion about which one you prefere.
> > 
> > 
> > I think the colors we are using now are not in line with the theme of the event. The purple color doesn't convey any sense. Also, a dark theme is a personal choice, but for a law theme, I think it doesn't make sense. But again, a dark theme is just a personal choice.
> 
> How about we keep the design as it is for the light mode, but add a button for people to toggle to a cool purple-themed dark mode? That way, everyone gets to choose what they like.

Seeing as purple left the color scheme of Jordi I believe it doesnt make sense anymore. Personally I also enjoyed purple before but being that the Marketing team decided to go with beje/cream and orange I believe the website should follow the same color scheme.

As it is right now I believe the website is not coherent with the rest of the event. If intend to keep this dark color scheme in the website I would suggest a way to change it to a light theme, such as a button, as it was mentioned previously. Other than that I believe most of the stuff have been said.

**Comment by GresosJ** [comment]
The final color scheme is here

File: src/components/Sponsors.jsx
Comment (pedrofp4444) [inline src/components/Sponsors.jsx]:
```suggestion
          Sponsorship
```

Comment (pedrofp4444) [inline src/components/Sponsors.jsx]: I don't know if you prefer ```Exclusive Sponsorship``` or only ```Sponsorship``` as it is.

Comment (GresosJ) [reply src/components/Sponsors.jsx]: Exclusive Sponsorship, sounds better

**Review by pedrofp4444 — APPROVED** [review]

**Review by pedrofp4444 — DISMISSED** [review]

File: src/components/Paper.jsx
Comment (tiago-bacelar) [inline src/components/Paper.jsx:33]: image
When seen in large responsiveness, the orange text is on top of the orange image, making it hard to read.

File: src/components/Sponsors.jsx
Comment (tiago-bacelar) [inline src/components/Sponsors.jsx]: "logol" is a bit scuffed ;) "logo_light" would be better. Could also be interesting to allow logo to be used by omission if no light logo is specified. There should also probably be a link to the sponsor's website

Comment (tiago-bacelar) [inline src/components/Sponsors.jsx]:
```suggestion
                  src={`/images/logos/${theme === 'dark' ? sponsor.logo : sponsor.logo_light ?? sponsor.logo}`}
```

File: src/components/Blog.jsx
Comment (tiago-bacelar) [inline src/components/Blog.jsx]:
```suggestion
        className="mx-auto bg-jordi-orange-500 text-white"
```
I think the button looks better like this :)

**Review by tiago-bacelar — CHANGES_REQUESTED** [review]
About the color scheme... Honestly, it's fine. I don't love it, but it is what it is, and we have to move on. The light/dark compromise seems fair enough. In my opinion, the dark mode looks better overall but is a bit funky in some places where the black background clashes with the black cards. Maybe it could be worth it to improve the contrast slightly somehow. Other than that, I don't think it's worth it to keep fiddling around.

File: src/components/Paper.jsx
Comment (GresosJ) [reply src/components/Paper.jsx:33]: Don't know who to improve this.

File: src/components/Header.jsx
Comment (MarioRodrigues10) [inline src/components/Header.jsx]: In order to remove this if statement that is duplicating code just do something like:

```c
  const icon = currentTheme === 'dark' ? <MoonIcon /> : <SunIcon />
```

**Review by MarioRodrigues10 — DISMISSED** [review]

File: src/components/Paper.jsx
Comment (tiago-bacelar) [reply src/components/Paper.jsx:33]: I'll try to figure smth out, I'll commit in a second...

File: src/components/Organization.jsx
Comment (tiago-bacelar) [inline src/components/Organization.jsx]:
```suggestion
  const { systemTheme, theme = 'dark' } = useTheme()
  const currentTheme = theme === 'system' ? systemTheme : theme
```

File: src/components/Sponsors.jsx
Comment (tiago-bacelar) [inline src/components/Sponsors.jsx]:
```suggestion
  const { systemTheme, theme = 'dark' } = useTheme()
  const currentTheme = theme === 'system' ? systemTheme : theme
```

File: src/components/Sponsors.jsx
Comment (tiago-bacelar) [inline src/components/Sponsors.jsx]:
```suggestion
                  currentTheme === 'dark'
```

File: src/components/Organization.jsx
Comment (tiago-bacelar) [inline src/components/Organization.jsx]:
```suggestion
               <Image
                    className="hover:opacity-80"
                    src={`/images/organization/${currentTheme === 'dark' ? organization.logo : organization.logo_light ?? organization.logo}`}
                    alt={organization.name}
                    height={80}
                    width={125}
                  />
```

**Review by tiago-bacelar — CHANGES_REQUESTED** [review]
When first loading into the page the theme is set as 'system'. This means whenever using the theme you must check this and assign it manually, as was already being done in Header

**Review by tiago-bacelar — DISMISSED** [review]

**Comment by MarioRodrigues10** [comment]
Just a small change, before you merge this could you use my SEI'23 photo instead of the one that you selected. Thanks :pray:

**Comment by GresosJ** [comment]
Wait a bit a need one photo, tomorrow morning i'll put it

**Comment by tiago-bacelar** [comment]
I just noticed I don't have a photo :sweat_smile: Could you please use my SEI'23 photo? :pray:
Also, not super important but Vision's logo in the footer isn't very visible in dark mode.

**Review by ruioliveira02 — COMMENTED** [review]
I would never approve of this in regular circumstances. However, JORDI starts on Monday and the site is still not deployed, so we do what we gotta do. (Also @MarioRodrigues10 and @tiago-bacelar said this was ok :sweat_smile:

**Review by ruioliveira02 — APPROVED** [review]

**Pull request merged**
